testsuite/src/it/java/org/graalvm/tests/integration/AppReproducersTest.java

```diff
@@ -467,12 +467,14 @@ public void imageioAWT(TestInfo testInfo, Apps app) throws IOException, Interrup
                     Files.writeString(Path.of(BASE_DIR, "..", DOCKER_GHA_SUMMARY_NAME), summary, UTF_8, CREATE, APPEND);
                 }
                 for (String base : RUNTIME_IMAGE_BASE) {
-                    if (DOCKER_GHA_SUMMARY_NAME != null) {
-                        if (BUILDER_IMAGE.contains("ubi9") && ("ubi8".equals(base) || "amzn2".equals(base))) {
+                    if (BUILDER_IMAGE.contains("ubi9") && ("ubi8".equals(base) || "amzn2".equals(base))) {
```

Comment:
Could we extract this as a function, please? Perhaps `isBuilderImageInCompatible()` or some such.

Reply:
+1

Reply:
That is the intention:  as per https://github.com/Karm/mandrel-integration-tests/pull/322 
```
Dockerfiles in general:

    Moved all to UBI9 runtime images until 

    https://github.com/Karm/mandrel-integration-tests/pull/313 is merged and then I'm gonna use its logic to run more runtime images.
```

Will do it here in this PR then :+1: